packages/core/src/createContext.ts

```diff
@@ -56,6 +58,22 @@ async function createContextByConfig(
   };
 }
 
+export function updateEnvironmentContext(
+  context: RsbuildContext,
+  configs: EnvironmentConfig[],
+) {
+  context.environments = configs.map((config) => ({
```

Comment:
`context.environments` can be an object, which is easier to use:

```js
// object
console.log(context.environments.web.entry);

// array
console.log(context.environments.find(item => item.name === 'web')?.entry);
```

Reply:
done